scripts/irods/test/test_iticket.py

```diff
@@ -766,3 +766,47 @@ def test_write_tickets_for_collections_allow_creation_of_data_objects__issue_591
                 self.admin.run_icommand(['ichmod', '-M', 'own', self.admin.username, fname])
 
             self.admin.run_icommand(['iticket', 'delete', ticket_name])
+
+    def test_ticket_create_with_relative_paths__issue_7103(self):
+        collection_1, collection_1_ticket_name = 'issue_7103_collection_1', 'collection_1_ticket'
+        collection_2, collection_2_ticket_name = 'issue_7103_collection_2', 'collection_2_ticket'
+        data_object_1, data_object_1_ticket_name = 'issue_7103_data_object_1', 'data_object_1_ticket'
+        data_object_2, data_object_2_ticket_name = 'issue_7103_data_object_2', 'data_object_2_ticket'
+        home_data_object, home_data_object_ticket_name = 'issue_7103_home_data_object', 'home_data_object_ticket'
+
+        home_collection_ticket_name = 'home_collection_ticket'
+
+        # Create collections
+        self.admin.assert_icommand(['imkdir', collection_1])
+        self.admin.assert_icommand(['imkdir', collection_2])
```

Comment:
Is this test class cleaning up the collections/data objects after it's done? Even when it fails?

Consider using the "session collection" for the active user which will always be cleaned up by the test framework. If that's not feasible due to the relative nature of the paths in the test, let's wrap the test with a try/finally block which cleans up the collections afterwards.

Reply:
I believe it is necessary to create some collections in order to demonstrate possible relative paths. I will add a try/finally block to remove the collections when done.